internal/extsvc/gitlab/client.go

```diff
@@ -184,7 +184,7 @@ func (p *ClientProvider) newClient(baseURL *url.URL, a auth.Authenticator, httpC
 	projCache := rcache.NewWithTTL(key, int(cacheTTL/time.Second))
 
 	rl := ratelimit.DefaultRegistry.Get(baseURL.String())
-	rlm := ratelimit.DefaultMonitorRegistry.GetOrSet(baseURL.String(), tokenHash, "", &ratelimit.Monitor{})
+	rlm := ratelimit.DefaultMonitorRegistry.GetOrSet(baseURL.String(), tokenHash, "core", &ratelimit.Monitor{})
```

Comment:
Maybe also "REST", since they also have a GraphQL API?
```suggestion
	rlm := ratelimit.DefaultMonitorRegistry.GetOrSet(baseURL.String(), tokenHash, "rest", &ratelimit.Monitor{})
```

Reply:
I went with core to be consistent with the GitHub.

Reply:
Oh, I thought you named that one "rest". NVM then!

Reply:
Sorry, you're right. I'll rename it. We're using 'core' in our metrics as this is what we already use in `github-proxy`.

Now that you've pointed out how confusing it is I think I'm going to rename it `rest` everywhere :)